addons/event/views/event_registration_views.xml

```diff
@@ -327,6 +353,29 @@
         </field>
     </record>
 
+    <record id="event_registration_action_stats_from_event" model="ir.actions.act_window">
+        <field name="name">Registration statistics</field>
+        <field name="res_model">event.registration</field>
+        <field name="view_mode">graph,pivot,kanban,tree,form</field>
+        <field name="domain">[('event_id', '=', active_id)]</field>
+        <field name="context">
+            {
+            'default_event_id': active_id,
+            'search_default_group_by_create_date_day': 1,
+            'search_default_status': 2,
+            'registration_view_hide_group_by_create_date_week': 1,
```

Comment:
And these should have an extra indentation level if we refer to the other one you modified.